composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "license": "MIT",
     "require": {
-        "laravel/framework": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*"
+        "laravel/framework": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*|5.5.*"
```

Comment:
@imjohnbon I would do package auto discovery as one PR. We merge it. Then do another PR. we test it against 5.5. If it works, I think we merge that too, then test again at launch.